set.mm

```diff
@@ -87504,32 +87529,59 @@ the first case of his notation (simple exponentiation) and subscript it
     $d A f $.
     $( The empty set is equinumerous only to itself.  Exercise 1 of
        [TakeutiZaring] p. 88.  (Contributed by NM, 27-May-1998.)  Avoid
-       ~ ax-pow .  (Revised by BTernaryTau, 31-Jul-2024.) $)
+       ~ ax-pow .  (Revised by BTernaryTau, 31-Jul-2024.)  Avoid ~ ax-un .
+       (Revised by BTernaryTau, 23-Sep-2024.) $)
     en0 $p |- ( A ~~ (/) <-> A = (/) ) $=
+      ( vf c0 cen wbr wceq cv wf1o wex cvv wcel wa wb encv breng syl ibi f1ocnv
+      ccnv 0ex f1o00 simprbi exlimiv f1oeq1 ceqsexv2d mp2an mpbir mpbiri impbii
+      f1o0 breq1 ) ACDEZACFZULACBGZHZBIZUMULUPULAJKCJKZLULUPMACNACBJJOPQUOUMBUO
+      CAUNSZHZUMACUNRUSURCFUMAURUAUBPUCPUMULCCDEZUTCCUNHZBIZVACCCHBCTCCUNCUDUJU
+      EUQUQUTVBMTTCCBJJOUFUGACCDUKUHUI $.
+    $( $j usage 'en0' avoids 'ax-pow' 'ax-un'; $)
+  $}
+
+  ${
+    $d A f $.
+    $( Obsolete version of ~ en0 as of 23-Sep-2024.  (Contributed by NM,
+       27-May-1998.)  Avoid ~ ax-pow .  (Revised by BTernaryTau, 31-Jul-2024.)
+       (Proof modification is discouraged.)  (New usage is discouraged.) $)
+    en0OLD $p |- ( A ~~ (/) <-> A = (/) ) $=
       ( vf c0 cen wbr wceq cv wf1o wex bren ccnv f1ocnv f1o00 simprbi syl sylbi
       exlimiv 0ex f1oeq1 f1o0 ceqsexv2d mpbir breq1 mpbiri impbii ) ACDEZACFZUF
       ACBGZHZBIUGACBJUIUGBUICAUHKZHZUGACUHLUKUJCFUGAUJMNOQPUGUFCCDEZULCCUHHZBIU
       MCCCHBCRCCUHCSTUACCBJUBACCDUCUDUE $.
-    $( $j usage 'en0' avoids 'ax-pow'; $)
   $}
 
   ${
     $d f A $.
     $( Obsolete version of ~ en0 as of 31-Jul-2024.  (Contributed by NM,
        27-May-1998.)  (Proof modification is discouraged.)
        (New usage is discouraged.) $)
-    en0OLD $p |- ( A ~~ (/) <-> A = (/) ) $=
+    en0OLDOLD $p |- ( A ~~ (/) <-> A = (/) ) $=
```

Comment:
Maybe we should keep this as ALT version now, because the proof is much shorter.

Reply:
Done in #4295